content/backstage/plugins/datadog.md

```diff
@@ -0,0 +1,153 @@
+---
+humanName: Datadog
+heading: 'Backstage Datadog plugin'
+lead: |
+  See your servers, your clouds, your metrics all in one place
+attribution:
+  text: Roadie
+  href: https://Roadie.io
+
+seo:
+  title: 'Backstage Datadog Plugin | Roadie'
+  description: |
+    See your servers, your clouds, your metrics all in one place
+
+logoImage: '../../assets/logos/datadog/datadog-logo-no-text.png'
+
+coverImage: '../../assets/datadog-plugin.png'
+coverImageAlt: 'A screenshot of the Datadog plugin.'
+
+gettingStarted:
+  - intro: With this plugin, you can embed Datadog graphs and dashboards into your instance of Backstage. Datadog is a monitoring service for cloud-scale applications, providing monitoring of servers, databases, tools, and services through a SaaS-based data analytics platform.
+
+  - title: Plugin installation
+
+  - intro: Install the Datadog plugin into Backstage.
+    language: bash
+    code: |
+      cd packages/app
+      yarn add @roadiehq/backstage-plugin-datadog
+
+  - intro: Add the Datadog plugin to the list of plugins.
+    language: typescript
+    code: |
+      // packages/app/src/plugins.ts
+      export { plugin as Datadog } from '@roadiehq/backstage-plugin-datadog';
+
+  - intro: Add a Datadog card to the overview tab of the EntityPage.tsx source file
+    language: typescript
+    code: |
+      // packages/app/src/components/catalog/EntityPage.tsx
+      import {
+            Router as DatadogRouter,
+            GraphWidget as DatadogGraphWidget,
+            isDatadogGraphAvailable as isDatadogWidgetAvailable,
+            } from '@roadiehq/backstage-plugin-datadog';
+
+      const ServiceEntityPage = ({ entity }: { entity: Entity }) => (
+        &lt;EntityPageLayout>
+            &lt;EntityPageLayout.Content
+                path="/datadog/*"
+                title="datadog"
+                element={<DatadogRouter entity={entity} />}
+                />
```

Comment:
The indentation on these lines is incorrect.

Reply:
Fixed and noted for the future